ISSUE_TEMPLATE: Create option for packages without a Hydra build status

khabdrick opened the pull request:
Resolves #387305

## Things done

Created an option for in ISSUE_TEMPLATE for build failure report for when a user reports an i686-linux-specific build failure.



- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

GaetanLepage committed:
box2d: move to pkgs/by-name

GaetanLepage committed:
box2d: 2.4.2 -> 3.1.0

Diff: https://github.com/erincatto/box2d/compare/v2.4.2...v3.1.0

Changelog: https://box2d.org/documentation/md_release__notes__v310.html

GaetanLepage committed:
qt6Packages.qmlbox2d: unstable-2022-08-25 -> 0-unstable-2024-04-15

FliegendeWurst committed:
fim: fix cross / strictDeps build

jvanbruegge committed:
isabelle: 2024 -> 2025

numinit committed:
fstar: 2024.09.05 -> 2025.03.25

The F* team significantly cleaned up the build. Based on the Nix
expression here:

https://github.com/FStarLang/FStar/blob/master/.nix/fstar.nix

Move to by-name and adopt the Z3 4.8.5 patches since we are now the only
package that depends on Z3 4.8.5 in nixpkgs.

numinit committed:
nixos/vwifi: init module

This module does not currently have its own test suite: it is intended
to be used in test suites for other modules that use wifi.

numinit committed:
nixos/kismet: init module

Use vwifi to write a proper test for Kismet. This test demonstrates how
to simulate wireless networks in NixOS tests, and extract meaningful
data by putting an interface in monitor mode using Kismet.

numinit committed:
kismet: add nixosTests passthru

numinit committed:
release-notes: add kismet and vwifi

numinit committed:
doc: init Testing Hardware Features section with vwifi

TomaSajt committed:
excalidraw_export: bump nan version to fix build

TomaSajt committed:
excalidraw_export: reenable darwin builds

jiriks74 committed:
min-ed-launcher: init at 0.11.3

- `min-ed-launcher` repository: https://github.com/rfvgyhn/min-ed-launcher

From project's description: Minimal Elite Dangerous Launcher

It's an alternative launcher for the game Elite: Dangerous. It has
provides a faster way to get into game and provides more functionality
than the official Frontier launcher.

Some of the advantages:
- Faster load times compared to the official launcher
- Auto-quit
- Can lauch (and quit) other programs (for interacting with the game)
  - For example EDMarketConnector https://github.com/EDCD/EDMarketConnector
- Can run Steam, Epic and Frontier Store accounts from a single game install
- Auto-restart

illdefined committed:
lib/types: check paths in pathWith with hasStorePathPrefix

This permits usage of content‐addressed derivations and has the added
benefit of checking normalised paths.

Aleksanaa committed:
media-downloader: refactor

illdefined committed:
lib/path: properly handle /. in hasStorePathPrefix

sumnerevans committed:
synapse-http-antispam: init at 0.3.0

https://github.com/maunium/synapse-http-antispam/releases/tag/v0.3.0

Co-authored-by: Ethan Carter Edwards <ethan@ethancedwards.com>
Signed-off-by: Sumner Evans <me@sumnerevans.com>

jvanbruegge committed:
isabelle-components.isabelle-linter: 2024-1.0.1 -> 2025-1.0.0

FliegendeWurst committed:
libndctl: fix musl build

Technically this patch will leak the first argument passed to daxctl.
Since it is a short-running CLI tool it doesn't matter much.

RossComputerGuy committed:
check-meta: add a teams attribute

RossComputerGuy committed:
docs: add new meta.teams

guylamar2006 committed:
honeycomb: Migrate to by-name

Fix scripts

Fix formatting.

run nixfmt on scripts.

qbisi committed:
adios2: init at 2.10.2

r-ryantm committed:
ultrastardx: 2025.3.0 -> 2025.4.0

r-ryantm committed:
apt: 2.9.35 -> 3.0.0

GaetanLepage committed:
python312Packages.pybind11-protobuf: 0-unstable-2024-11-01 -> 0-unstable-2025-02-10

Co-authored-by: Someone Serge <else@someonex.net>

MattSturgeon committed:
lib.options.mkPackageOption: use lib.showAttrPath

Make use of `lib.showAttrPath` instead of manually doing `concatStringsSep "."`.

This means edge-cases such as the attr-path including names that are not
valid nix identifiers will be handled better.

See:
- https://nix.dev/manual/nix/2.26/language/identifiers
- https://nixos.org/manual/nixpkgs/unstable/#function-library-lib.attrsets.showAttrPath

MattSturgeon committed:
lib/tests/modules: test all `mkPackageOption` cases

There were several test case options declared in `declare-mkPackageOption.nix`
that were not actually tested in `modules.sh`.

RossComputerGuy committed:
ci/eval/compare: add meta.teams

numinit committed:
maintainers/scripts: add get-maintainer-pings-between.sh

Sometimes it is useful to see which maintainers your change will ping,
to know who you are going to spam in advance.

numinit committed:
ci/eval/compare: support optional byName argument

Sometimes it is quite useful to output names instead of GitHub IDs, e.g.
for maintainer scripts that show you who you would ping. Add this as an
option, but keep the existing default.

qbisi committed:
pocl: fix icd support

Enable icd support requires both ocl-icd and opencl-headers.

qbisi committed:
python312Packages.pyopencl: doCheck with pocl

reckenrode committed:
solana-cli: get libc++ headers on Darwin from stdenv.cc.libcxx

reckenrode committed:
python3Packages.py3exiv2: get libc++ headers on Darwin from stdenv.cc.libcxx

reckenrode committed:
bazel_{5,6,7}: get libc++ headers on Darwin from stdenv.cc.libcxx

reckenrode committed:
moltenvk: get libc++ headers on Darwin from stdenv.cc.libcxx

reckenrode committed:
ocamlPackages.eigen: get libc++ headers on Darwin from stdenv.cc.libcxx

reckenrode committed:
python3Packages.datatable: get libc++ headers on Darwin from stdenv.cc.libcxx

reckenrode committed:
python3Packages.kiwisolver: get libc++ headers on Darwin from stdenv.cc.libcxx

reckenrode committed:
fh: get libc++ headers on Darwin from stdenv.cc.libcxx

reckenrode committed:
servo: get libc++ headers on Darwin from stdenv.cc.libcxx

reckenrode committed:
python3Packages.numba: get libc++ headers on Darwin from stdenv.cc.libcxx

reckenrode committed:
R: get libc++ headers on Darwin from stdenv.cc.libcxx

reckenrode committed:
python3Packages.kivy: get libc++ headers on Darwin from stdenv.cc.libcxx

r-ryantm committed:
nextcloud-talk-desktop: 1.1.5 -> 1.1.6

siriobalmelli committed:
tsduck: 3.31-2761 -> 3.40-4165; fix Darwin build

Update 3.31-2761 -> 3.40-4165.

Fix build for Darwin.

Add librist.

Remove 'rec' derivation antipattern.

Minor stylistic cleanups.

Signed-off-by: Sirio Balmelli <sirio@b-ad.ch>

qbisi committed:
pocl: add setupHook to specifies directory to scan for ICDs

Adds a setup hook that sets the OCL_ICD_VENDORS environment variable
to point to a directory containing ICD (Installable Client Driver) files.

These files define the available OpenCL backends, allowing other OpenCL
applications to detect and use the appropriate drivers in sandbox environment.

FliegendeWurst committed:
python312Packages.mkdocs: fix cross build

setuptools is required unconditionally as tool.hatch.build.hooks.custom

r-ryantm committed:
super-productivity: 12.0.3 -> 12.0.5

qbisi committed:
python312Packages.pyopencl: support Darwin platform

rhendric committed:
nixos/pam: conditional enabling of services

mjm committed:
python3Packages.rq: fix failing tests

wegank committed:
python312Packages.pybids: 0.18.1 -> 0.19.0

wegank committed:
python312Packages.bidsschematools: 1.0.4 -> 1.0.5

qbisi committed:
pocl: add installCheckPhase

qbisi committed:
pocl: support Darwin platform

qbisi committed:
pocl: setting KERNELLIB_HOST_CPU_VARIANTS=distro on x86_64 only

kirillrdy committed:
awsebcli: 3.21 -> 3.22

diff: https://github.com/aws/aws-elastic-beanstalk-cli/compare/3.21...3.22

Aleksanaa committed:
clapper-unwrapped: separated from clapper

Aleksanaa committed:
clapper-enhancers: init at 0.8.2

Aleksanaa committed:
clapper: rewrite as a wrapper

Aleksanaa committed:
treewide: use clapper-unwrapped instead of clapper

GaetanLepage committed:
python312Packages.jax[lib]: 0.5.3 -> 0.6.0

Diff: https://github.com/jax-ml/jax/compare/jax-v0.5.3...jax-v0.6.0

Changelog: https://github.com/jax-ml/jax/blob/jax-v0.6.0/CHANGELOG.md

qbisi committed:
superlu_dist: add fortran support for Darwin platform

r-ryantm committed:
senpai: 0.4.0 -> 0.4.1

GaetanLepage committed:
python312Packages.dm-haiku: adapt to jax 0.6.0 API changes

https://docs.jax.dev/en/latest/jax.extend.html
https://github.com/google-deepmind/dm-haiku/commit/cfe8480d253a93100bf5e2d24c40435a95399c96

GaetanLepage committed:
python312Packages.waymax: adapt to jax 0.6.0 new API changes

https://github.com/waymo-research/waymax/pull/77

GaetanLepage committed:
python312Packages.waymax: fix version

provokateurin committed:
nextcloud29: 29.0.15 -> 29.0.16

provokateurin committed:
nextcloud30: 30.0.9 -> 30.0.10

provokateurin committed:
nextcloud31: 31.0.3 -> 31.0.4

GaetanLepage committed:
python312Packages.distrax: adapt to jax 0.6.0 new API changes

GaetanLepage committed:
python312Packages.gymnasium: skip failing tests

https://github.com/Farama-Foundation/Gymnasium/issues/1360
https://github.com/Farama-Foundation/Gymnasium/pull/1361

marcin-serwin committed:
SDL2_image: add option to build without STB backend

marcin-serwin committed:
pygame{,-ce}: use latest version of SDL2_image

r-ryantm committed:
rain: 1.22.0 -> 1.23.0

mitchmindtree committed:
puredata: fix pd-gui by providing tk path

Prior to this commit, the `pd-gui` command fails to launch with:

`pd-gui.tcl: line 3: exec: wish: not found`

This is because we were only wrapping `pd` with the `tk` path, but not
`pd-gui`.

This also resulted in the puredata desktop shortcut failing, as it
attempts to invoke `pd-gui` directly.

This commit fixes these issues by also wrapping `pd-gui` with the `tk`
path.

Commit:
chatbox: fix update script

Commit:
chatbox: 1.11.3 -> 1.11.12

antonmosich committed:
jrnl: fix build

python3Packages.rich was updated to 14.0.0 in #395156, and jrnl requires
rich < 14, but the new version did not break anything.

qbisi committed:
pocl: migrate to lib.cmakeBool and lib.cmakeFeature

r-ryantm committed:
linuxPackages.hid-tmff2: 0.82-unstable-2025-02-03 -> 0-unstable-2025-04-12

r-ryantm committed:
minimap2: 2.28 -> 2.29

r-ryantm committed:
helm-ls: 0.2.0 -> 0.2.1

r-ryantm committed:
jbrowse: 3.2.0 -> 3.3.0

r-ryantm committed:
ghidra-extensions.kaiju: 250220 -> 250417

r-ryantm committed:
audiowaveform: 1.10.1 -> 1.10.2

qbisi committed:
pocl: fix description

keenanweaver committed:
supermodel: 0-unstable-2024-11-07 -> 0-unstable-2025-04-17, fix libGL

Adds libGL back after change in https://github.com/NixOS/nixpkgs/pull/393386

r-ryantm committed:
lemminx: 0.30.0 -> 0.30.1

phaer committed:
php.extensions.excimer: init at 1.2.3

r-ryantm committed:
eigenmath: 338-unstable-2025-04-03 -> 338-unstable-2025-04-18

r-ryantm committed:
ptcollab: 0.6.4.8 -> 0.6.4.9

r-ryantm committed:
chirp: 0.4.0-unstable-2025-04-01 -> 0.4.0-unstable-2025-04-17

Commit:
charles4: 4.6.4 -> 4.6.8

Commit:
charles5: init at 5.0

r-ryantm committed:
python312Packages.tskit: 0.6.1 -> 0.6.2

mweinelt committed:
python313Packages.voluptuous-openapi: 0.0.6 -> 0.0.7

https://github.com/home-assistant-libs/voluptuous-openapi/releases/tag/v0.0.7

r-ryantm committed:
lunacy: 11.2.1 -> 11.3

qbisi committed:
pocl: remove updateScript

r-ryantm committed:
dracula-theme: 4.0.0-unstable-2025-04-01 -> 4.0.0-unstable-2025-04-18

r-ryantm committed:
terraform-providers.google: 6.29.0 -> 6.30.0

Commit:
clockify: 2.2.9.2224 -> 2.3.2.2633

r-ryantm committed:
yamlscript: 0.1.95 -> 0.1.96

r-ryantm committed:
python312Packages.art: 6.4 -> 6.5

r-ryantm committed:
python312Packages.langfuse: 2.60.2 -> 2.60.3

r-ryantm committed:
docker-credential-gcr: 2.1.27 -> 2.1.28

r-ryantm committed:
terraform-providers.tencentcloud: 1.81.181 -> 1.81.183

r-ryantm committed:
mpvScripts.visualizer: 0-unstable-2024-09-26 -> 0-unstable-2025-04-12

paveloom committed:
streamrip: relax all dependencies

r-ryantm committed:
treesheets: 0-unstable-2025-04-05 -> 0-unstable-2025-04-14

r-ryantm committed:
rqlite: 8.36.16 -> 8.36.17

linsui committed:
goldendict-ng: 25.02.0 -> 25.05.0

linsui committed:
goldendict-ng: add linsui to maintainer

antonmosich committed:
notmuch: 0.38.3 -> 0.39

Importing notmuch2 in the test suite is still broken, see
78529f2949a38d2bc902c119caf27de909c5f784 for more information.

Co-authored-by: Damien Cassou <damien@cassou.me>

Aleksanaa committed:
libseccomp: remove loongarch64-linux from badPlatforms

kirillrdy committed:
golangci-lint-langserver: re-enable check

r-ryantm committed:
pixelflasher: 7.11.3.0 -> 7.11.4.0

Guanran928 committed:
sing-box: 1.11.7 -> 1.11.8

Diff: https://github.com/SagerNet/sing-box/compare/v1.11.7...v1.11.8

r-ryantm committed:
updatecli: 0.97.0 -> 0.98.0

qbisi committed:
superlu_dist: use pkg-config to find LAPACK

CMake in Nixpkgs may mistakenly detect BLAS or the Apple SDK
framework when locating the LAPACK directory.
This issue can be resolved by setting BLA_PREFER_PKGCONFIG=ON.

r-ryantm committed:
nb: 7.16.1 -> 7.17.0

r-ryantm committed:
python312Packages.pyoverkiz: 1.16.5 -> 1.17.0

r-ryantm committed:
gapless: 4.3.1 -> 4.4

r-ryantm committed:
qlog: 0.43.0 -> 0.43.1

devusb committed:
python3Packages.flask-limiter: fix building with latest rich

r-ryantm committed:
coroot: 1.9.10 -> 1.10.1

sikmir committed:
qlog: migrate to by-name

r-ryantm committed:
terraform-providers.tfe: 0.64.0 -> 0.65.1

qbisi committed:
kahip: init at 3.18

r-ryantm committed:
civo: 1.2.0 -> 1.2.1

r-ryantm committed:
mailpit: 1.24.0 -> 1.24.1

SigmaSquadron commented on .github/ISSUE_TEMPLATE/04_build_failure.yml:
```diff
@@ -60,13 +60,15 @@ body:
         - If there's a <img src="https://raw.githubusercontent.com/NixOS/hydra/refs/heads/master/src/root/static/images/emojione-check-2714.svg" width="20px" align="top" alt="Green Check Mark"> icon near the package entry, then it most likely means it's a local issue with your system. (Maybe you ran out of space?)
           You can still open a build failure report, but please say '**No, Hydra cannot reproduce this build failure.**' below.
         - If there's a <img src="https://raw.githubusercontent.com/NixOS/hydra/refs/heads/master/src/root/static/images/emojione-question-2754.svg" width="20px" align="top" alt="Gray Question Mark"> icon near the package entry, say '**Hydra is currently rebuilding this package.**'
+        - If you are using a system with i686 (32 bits x86) architecture, e.g, Pentium II, III, and some early Pentium 4, say '**Hydra doesn’t try to build the package.**'
```
`i386` packages aren't the only reason why Hydra would forgo building packages. I recommend changing this line to include all packages skipped by Hydra with more generic wording.

Aleksanaa replied:
```suggestion
        - If hydra isn't supposed to build the package at all, say '**Hydra doesn’t try to build the package.**'
```

r-ryantm committed:
gearlever: 3.0.2 -> 3.2.2

drupol committed:
doc: use `writableTmpDirAsHomeHook`

r-ryantm committed:
roslyn-ls: 4.14.0-3.25164.3 -> 5.0.0-1.25204.1

drupol committed:
doc: add missing phase hooks

drupol committed:
doc: use `finalAttrs` pattern

drupol committed:
doc: remove useless `rec`

drupol committed:
doc: replace `rev` with `tag`

drupol committed:
doc: do not reuse `pname`

FKouhai committed:
base16-schemes: unstable-2024-11-12 -> unstable-2025-04-18

alexfmpe committed:
redu: init at 0.2.12

herbetom committed:
borgbackup: 1.4.0 -> 1.4.1

Changelog: https://github.com/borgbackup/borg/blob/1.4.1/docs/changes.rst#version-141-2025-04-19

GaetanLepage committed:
python312Packages.rlax: fetch patches to fix broken tests

mweinelt committed:
weechat-unwrapped: add update script

mweinelt committed:
weechat-unwrapped: 4.5.1 -> 4.6.2

https://github.com/weechat/weechat/releases/tag/v4.5.2
https://github.com/weechat/weechat/releases/tag/v4.6.0
https://github.com/weechat/weechat/releases/tag/v4.6.1
https://github.com/weechat/weechat/releases/tag/v4.6.2

mweinelt committed:
weechat-unwrapped: fix changelog url

The changelog on the homepage cannot be computed since it contains the
release date now. The GitHub releases contain the same information, so
we're good.

mweinelt committed:
weechat-unwrapped: refactor

r-ryantm committed:
docker-compose: 2.35.0 -> 2.35.1

Commit:
copier: 9.4.1 -> 9.6.0

Diff: https://github.com/copier-org/copier/compare/refs/tags/v9.4.1...refs/tags/v9.6.0

Changelog: https://github.com/copier-org/copier/blob/v9.6.0/CHANGELOG.md

r-ryantm committed:
vulkan-caps-viewer: 4.00 -> 4.01

r-ryantm committed:
fq: 0.14.0 -> 0.15.0

r-ryantm committed:
enzyme: 0.0.173 -> 0.0.174

r-ryantm committed:
cpp-utilities: 5.28.0 -> 5.28.1

numinit committed:
nixos/mattermost: correct file upload directory

Fix compatibility with previous versions by making sure all the uploads
and plugins end up in the correct directory. Add a test for the exact
path we care about to ensure that it doesn't work "on accident."

Discovered while updating instances to unstable.

r-ryantm committed:
mieru: 3.13.1 -> 3.14.0

r-ryantm committed:
python312Packages.neo: 0.14.0 -> 0.14.1

numinit committed:
release-notes: update Mattermost release notes

Aleksanaa committed:
rust-streamdeck: remove Cargo.lock

r-ryantm committed:
python312Packages.clarifai-grpc: 11.3.1 -> 11.3.2

r-ryantm committed:
terraform-providers.equinix: 3.5.0 -> 3.6.0

booxter committed:
fpc: add apple-sdk SDKROOT to default library paths

This fixes darwin build for xidel, among other things.

Note: there are some impurities in the package where xcode and other
paths may be injected into build environment. These are left out of
scope for this patch, to be dealt by other interested parties.

Signed-off-by: Ihar Hrachyshka <ihar.hrachyshka@gmail.com>

r-ryantm committed:
openrefine: 3.9.2 -> 3.9.3

r-ryantm committed:
felix-fm: 2.16.0 -> 2.16.1

dotlambda committed:
home-assistant: 2025.4.2 -> 2025.4.3

Diff: https://github.com/home-assistant/core/compare/refs/tags/2025.4.2...2025.4.3

Changelog: https://github.com/home-assistant/core/releases/tag/2025.4.3

Co-Authored-By: Martin Weinelt <hexa@darmstadt.ccc.de>

dotlambda committed:
python313Packages.bellows: 0.44.0 -> 0.44.1

Diff: https://github.com/zigpy/bellows/compare/refs/tags/0.44.0...0.44.1

Changelog: https://github.com/zigpy/bellows/releases/tag/0.44.1

dotlambda committed:
python313Packages.zigpy: 0.78.0 -> 0.78.1

Diff: https://github.com/zigpy/zigpy/compare/refs/tags/0.78.0...0.78.1

Changelog: https://github.com/zigpy/zigpy/releases/tag/0.78.1

dotlambda committed:
python313Packages.zha: 0.0.55 -> 0.0.56

Diff: https://github.com/zigpy/zha/compare/refs/tags/0.0.55...0.0.56

Changelog: https://github.com/zigpy/zha/releases/tag/0.0.56

dotlambda committed:
python313Packages.homeassistant-stubs: 2025.4.2 -> 2025.4.3

Diff: https://github.com/KapJI/homeassistant-stubs/compare/refs/tags/2025.4.2...2025.4.3

Changelog: https://github.com/KapJI/homeassistant-stubs/releases/tag/2025.4.3

r-ryantm committed:
python312Packages.weblate-schemas: 2024.2 -> 2025.1

numinit committed:
mattermost: disable more broken tests

r-ryantm committed:
terraform-providers.acme: 2.31.0 -> 2.32.0

Guanran928 committed:
fcitx5-pinyin-zhwiki: 20240909 -> 20250415

r-ryantm committed:
golangci-lint-langserver: 0.0.10 -> 0.0.11

khabdrick commented:
@SigmaSquadron does it look good now?

SigmaSquadron commented:
Sorry for the nit, but this is publicly shown to all users and contributors. 'Hydra' needs to be capitalised.

wegank committed:
revolver: fix update script

wegank committed:
igvm-tooling: fix update script

fabaff committed:
python313Packages.cyclopts: 3.13.0 -> 3.13.1

Diff: https://github.com/BrianPugh/cyclopts/compare/refs/tags/v3.13.0...refs/tags/v3.13.1

Changelog: https://github.com/BrianPugh/cyclopts/releases/tag/v3.13.1

fabaff committed:
python313Packages.identify: 2.6.9 -> 2.6.10

Diff: https://github.com/pre-commit/identify/compare/refs/tags/v2.6.9...refs/tags/v2.6.10

wegank committed:
wayclip: fix update script

r-ryantm committed:
opkssh: 0.3.0 -> 0.5.1

DimitarNestorov committed:
daisydisk: 4.30 -> 4.31

r-ryantm committed:
glamoroustoolkit: 1.1.19 -> 1.1.21

fabaff committed:
cpuid: 20241023 -> 20250419

wolfgangwalther committed:
teams/haskell: add wolfgangwalther

K900 committed:
linux_testing: 6.15-rc1 -> 6.15-rc2

K900 committed:
linux_6_14: 6.14.2 -> 6.14.3

K900 committed:
linux_6_13: 6.13.11 -> 6.13.12

K900 committed:
linux_6_12: 6.12.23 -> 6.12.24

r-ryantm committed:
automatic-timezoned: 2.0.72 -> 2.0.73

trofi committed:
libofa: fix patch application

Without the change patches fail to apply as `$patches` contain compressed
tarball:

    applying patch /nix/store/yrl3bq4ag3787yq32zawxnd6nsyk4a8r-libofa_0.9.3-5.debian.tar.gz
    (Stripping trailing CRs from patch; use --binary to disable.)
    patching file lib/signal_op.cpp
    (Stripping trailing CRs from patch; use --binary to disable.)
    patching file examples/example.cpp
    (Stripping trailing CRs from patch; use --binary to disable.)

With `patch-2.8` it will become an error.

K900 committed:
home-assistant-custom-lovelace-modules.universal-remote-card: 4.3.12 -> 4.4.0

Diff: https://github.com/Nerwyn/android-tv-card/compare/4.3.12...4.4.0

LordGrimmauld committed:
aonsoku: remove incorrect duplicate libsoup dependency

LordGrimmauld committed:
neohtop: remove incorrect duplicate libsoup dependency

LordGrimmauld committed:
rclone-ui: remove incorrect duplicate libsoup dependency

LordGrimmauld committed:
yaak: remove incorrect duplicate libsoup dependency

r-ryantm committed:
python312Packages.django-cacheops: 7.1 -> 7.2

r-ryantm committed:
prometheus-zfs-exporter: 2.3.6 -> 2.3.8

erictapen committed:
weblate: relax constraints on weblate-schemas

marcin-serwin committed:
SDL2_image_2_0: drop

marcin-serwin committed:
pygame-ce: unbreak

mweinelt committed:
home-assistant.python.pkgs.pytest-homeassistant-custom-component: 0.13.233 -> 0.13.235

https://github.com/MatthewFlamm/pytest-homeassistant-custom-component/blob/refs/tags/0.13.235/CHANGELOG.md

doronbehar committed:
mathematica: 14.2.0 -> 14.2.1

hatch01 committed:
kalker: fix build by patching cargo lock

r-ryantm committed:
exportarr: 2.1.0 -> 2.2.0

corngood committed:
godot: disable autoPatchelf on $debug

Fixes: #399818

GaetanLepage committed:
box2d_2: init at 2.4.2

GaetanLepage committed:
libreoffice: use box2d_2

spacedentist committed:
spr: update GitHub url and maintainer's GitHub account

I am the creator and maintainer of spr. I started the project while
employed at Cord (who at the time owned the cord.com domain), and Cord
allowed me to spend some time at work maintaining spend. I have
since left Cord, and later Cord ceased operations. The spr repository
was then transferred to my personal GitHub account.

This commit updates the GitHub URL of spr, and also changes the
maintainer from my Cord-related GitHub account (sven-of-cord) to my
personal one (spacedentist).

mweinelt committed:
python313Packages.env-canada: fix the 0.10.1 update

The update did not bump the src fod hahs.

marcin-serwin committed:
SDL2_image: remove unused deps

sikmir committed:
qlog: fix installation on darwin

r-ryantm committed:
monkeysAudio: 11.05 -> 11.08

diogotcorreia committed:
ultrastardx: add diogotcorreia as maintainer

GaetanLepage committed:
protobuf_23: drop

r-ryantm committed:
bazel-watcher: 0.25.3 -> 0.26.0

josh committed:
catppuccin: add alacritty source

dotlambda committed:
python313Packages.sentry-sdk: 1.45.1 -> 2.25.0

drupol committed:
orthanc-framework: remove `outputs` attribute

drupol committed:
orthanc-plugin-dicomweb: remove custom patch, cleanup

mweinelt committed:
home-assistant-custom-components.daikin_onecta: 4.2.3 -> 4.2.6

https://github.com/jwillemsen/daikin_onecta/tag/v4.2.6

mweinelt committed:
home-assistant-custom-components.midea_ac: 2025.3.1 -> 2025.4.0

https://github.com/mill1000/midea-ac-py/compare/refs/tags/2025.3.1...refs/tags/2025.4.0

mweinelt committed:
home-assistant-custom-components.moonraker: 1.7.0 -> 1.7.1

https://github.com/marcolivierarsenault/moonraker-home-assistant/releases/tag/1.7.1

mweinelt committed:
home-assistant-custom-components.tuya_local: 2025.3.0 -> 2025.4.0

https://github.com/make-all/tuya-local/releases/tag/2025.4.0

mweinelt committed:
home-assistant-custom-components.xiaomi_miot: 1.0.16 -> 1.0.17

https://github.com/al-one/hass-xiaomi-miot/releases/tag/v1.0.17

r-ryantm committed:
nezha-theme-nazhua: 0.6.4 -> 0.6.6

mweinelt committed:
pretix: drop geoip2 override

dotlambda committed:
python313Packages.proton-vpn-network-manager: mark broken

dotlambda committed:
python313Packages.sentry-sdk_1: drop

dotlambda committed:
python313Packages.sentry-sdk_2: turn into alias

r-ryantm committed:
vimPlugins.cmp-async-path: 0-unstable-2024-10-21 -> 0-unstable-2025-04-13

Aleksanaa committed:
eduli: fix hash

r-ryantm committed:
firezone-gateway: 1.4.5 -> 1.4.6

r-ryantm committed:
firezone-headless-client: 1.4.5 -> 1.4.6

r-ryantm committed:
nwg-dock-hyprland: 0.4.4 -> 0.4.5

Aleksanaa commented:
Please squash into one commit. If you are unable to do that, I can squash merge it, but it would be better to do it yourself.

kalekseev committed:
ruff-lsp: remove

The package was deprecated in February 2025 and is not updated to support
latest version of ruff.

r-ryantm committed:
postgresqlPackages.timescaledb-apache: 2.19.2 -> 2.19.3

drupol committed:
hedgedoc: use `writableTmpDirAsHomeHook`

r-ryantm committed:
krillinai: 1.1.0 -> 1.1.3

GaetanLepage committed:
python312Packages.pytensor: fix hash

r-ryantm committed:
cargo-mobile2: 0.17.6 -> 0.20.0

r-ryantm committed:
trilium-next-{desktop,server}: 0.92.7 -> 0.93.0 (#400419)

r-ryantm committed:
python312Packages.python-codon-tables: 0.1.13 -> 0.1.15

r-ryantm committed:
timewarrior: 1.7.1 -> 1.8.0

acid-bong committed:
nh: make use of finalAttrs

acid-bong committed:
nh, nixos/doc: update repo owner (viperML -> nix-community)

acid-bong committed:
nh: allow generating completions while cross-compiling

Kek5chen committed:
maintainers: kekschen -> willow

drupol committed:
winbox4: 4.0beta16 -> 4.0beta18

khabdrick commented:
Okay.
I will do that is some time.

khabdrick commented:
@Aleksanaa I think I have made a very big mistake on this branch
